packages/styles/src/variables/_elevation.scss

```diff
@@ -0,0 +1,36 @@
+@use 'sass:map';
+
+// Values are kept in PX for easier copy-ability from figma and better overview, rems
+// would provide little value added
+$elevation-1: 0.25px 1px 1px rgba(0, 0, 0, 0.1), 1px 2px 2.5px 1px rgba(0, 0, 0, 0.1);
+$elevation-2: 0.25px 1.5px 2px rgba(0, 0, 0, 0.1), 2px 2px 5px 2px rgba(0, 0, 0, 0.11);
+$elevation-3: 0.5px 3px 4px rgba(0, 0, 0, 0.1), 2px 4px 10px 4px rgba(0, 0, 0, 0.12);
+$elevation-4: 0.75px 4.5px 6px rgba(0, 0, 0, 0.1), 2px 6px 15px 6px rgba(0, 0, 0, 0.13);
+$elevation-5: 1px 6px 8px rgba(0, 0, 0, 0.1), 2px 8px 20px 8px rgba(0, 0, 0, 0.14);
+$elevation-6: 0.25px 1px 1px rgba(var(--post-shadow-color-rgb), 0.4),
+  1px 2px 2.5px 1px rgba(var(--post-shadow-color-rgb), 0.4);
+$elevation-7: 0.25px 1.5px 2px rgba(var(--post-shadow-color-rgb), 0.4),
+  2px 2px 5px 2px rgba(var(--post-shadow-color-rgb), 0.41);
+$elevation-8: 0.5px 3px 4px rgba(var(--post-shadow-color-rgb), 0.4),
+  2px 4px 10px 4px rgba(var(--post-shadow-color-rgb), 0.42);
+$elevation-9: 0.75px 4.5px 6px rgba(var(--post-shadow-color-rgb), 0.4),
+  2px 6px 15px 6px rgba(var(--post-shadow-color-rgb), 0.43);
+$elevation-10: 1px 6px 8px rgba(var(--post-shadow-color-rgb), 0.4),
+  2px 8px 20px 8px rgba(var(--post-shadow-color-rgb), 0.44);
```

Comment:
In my eyes it would make sense, to name the colored elevations differently and to create an own map for them:
```scss
$elevation-1: ...

$elevation-colored-1: ...

$elevation-map: () !default;
$elevation-map: map.merge(
  $elevation-map,
  (
    'elevation-1': $elevation-1,
    ...
    'elevation-5': $elevation-5,
);

$elevation-colored-map: () !default;
$elevation-colored-map: map.merge(
  $elevation-colored-map,
  (
    'elevation-colored-1': $elevation-colored-1,
    ...
    'elevation-colored-5': $elevation-colored-5,
);
```

Reply:
The distinction would also make sense for class names since at the moment the `.elevation-5` shadow looks larger than the `.elevation-6` on light backgrounds.

But don't you think you should choose only one type of shadow? Keeping both seems quite confusing to me, as it stands I wouldn't know when to use one or the other

Reply:
@alizedebray This is a good argument. I would also not know when to use black or colored elevations and probably just try which one looks better to me in every single case.